docs/checkers/index.md

```diff
@@ -2654,6 +2654,27 @@ Corrected version:
 print("Hello World!")  # This file ends with a single newline character! :)
 ```
 
+(W2301)=
+
+### Unnecessary ellipsis (W2301)
```

Comment:
This is good, but the positioning of the message could be better. Please move this new entry to right after the "unnecessary pass" entry, since these errors are very closely related.